### ISSUE TITLE
What is a reasonable of InstructBLIP fine-tuning?

### ISSUE THREAD

**Issue by Richar-Du** [salesforce/LAVIS#493]
I want to fine-tune InstructBLIP on a hold-in dataset, such as LLaVA. What is a reasonable loss? I observed that the training loss does not decrease.

```
Train: data epoch: [0]  [   0/5574]  eta: 13:10:52  lr: 0.000000  loss: 1.1933  time: 8.5131  data: 0.0000  max mem: 59680
2023-08-17 10:02:32,829 [INFO] Reducer buckets have been rebuilt in this iteration.
Train: data epoch: [0]  [  50/5574]  eta: 2:00:17  lr: 0.000001  loss: 1.0801  time: 1.1755  data: 0.0000  max mem: 63010
Train: data epoch: [0]  [ 100/5574]  eta: 1:54:28  lr: 0.000001  loss: 1.1740  time: 1.1968  data: 0.0000  max mem: 63010
Train: data epoch: [0]  [ 150/5574]  eta: 1:50:49  lr: 0.000002  loss: 1.1129  time: 1.1954  data: 0.0000  max mem: 63010
Train: data epoch: [0]  [ 200/5574]  eta: 1:48:30  lr: 0.000002  loss: 1.1627  time: 1.1997  data: 0.0000  max mem: 63218
```

**Comment by yPanStupidog** [comment]
How to fine-tune InstructBLIP? Should I use the train.py again? Thanks for your help!

**Comment by linzhiqiu** [comment]
Have you solved the issue? I tried to replicate BLIP-2 second-stage pre-training, but my loss is also not decreasing.

**Comment by ShuxunoO** [comment]
How can you finetune the BLIP or ALBEL model, can you please give me a reference？ please！
